docker-compose.yml

```
@@ -0,0 +1,34 @@
+services:
+  postgres:
+    image: postgres:latest
+    restart: always
+    shm_size: 128mb
+    container_name: postgres
+    environment:
+      POSTGRES_USER: ${DATABASE_USER}
+      POSTGRES_PASSWORD: ${DATABASE_PASSWORD}
+      POSTGRES_DB: ${DATABASE_NAME}
+    ports:
+      - "5433:5432"
```

lets move it to the end.
```suggestion
  postgres:
    image: postgres:alpine3.18
    container_name: postgres
    healthcheck:
      test: ["CMD", "pg_isready", "-U", "postgres", "-d", ${DATABASE_NAME} ]
      interval: 5s
      timeout: 5s
      retries: 5
    environment:
      POSTGRES_USER: ${DATABASE_USER}
      POSTGRES_PASSWORD: ${DATABASE_PASSWORD}
      POSTGRES_DB: ${DATABASE_NAME}
    expose:
      - 5433
```
- With expose, port wont be accessible from outside docker.
- alpine image is lighter
- we need to add command to check if db is healthy